common/types/observability_saved_object_attributes.ts

```diff
@@ -9,6 +9,7 @@ import { SavedQuery, SavedVisualization } from './explorer';
 export const VISUALIZATION_SAVED_OBJECT = 'observability-visualization';
 export const SEARCH_SAVED_OBJECT = 'observability-search';
 export const NOTEBOOK_SAVED_OBJECT = 'observability-notebook';
+export const NOTEBOOK_SAVED_OBJECT_PATH = 'observability-notebooks';
```

Comment:
this is https://github.com/opensearch-project/dashboards-observability/blob/d912d7b174190242a8fd5dde0281ae927326a487/common/constants/shared.ts#L68?